src/app/config/translations/en.ts

```diff
@@ -296,6 +296,38 @@ export const locale = {
             UNSUITABLE: { name: 'Unsuitable', cssColorClass: 'nhsuk-tag--red' },
             WITHDRAWN: { name: 'Withdrawn', cssColorClass: 'nhsuk-tag--red' },
             COMPLETE: { name: 'Completed', cssColorClass: 'nhsuk-tag--dark-grey' }
+          },
+          grouped_status: {
+            RECORD_NOT_SHARED: {
+              name: 'Record not shared',
+              description: 'You have not yet shared your innovation record for a needs assessment review.',

```

Comment:
Is this description for an specific user type? Call me please to share some ideas...